### PR TITLE
Css transition

### PR DESCRIPTION
**Pull request by alejobh**
## Summary
A CSS Transition has been added to the Tech card.

## Screenshots
Alt text

## Trello Card
https://trello.com/c/AHGq7QwF

### CODE REVIEW

File: maquetado/application.css [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+  margin: 30px 50px; }
+
+.techs-container {
+  display: flex;
+  flex-flow: wrap;
+  margin: 30px 140px; }
+
+body {
+  font-family: "Roboto", sans-serif; }
+
```
Comment (henryzarza) [inline maquetado/application.css]: Add this file the `gitignore`

File: maquetado/scss/tech.scss [hunk 2]
```diff
@@ -1,5 +1,5 @@
-@import 'variables';
-@import 'margins';
+@import "variables";
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]: Remember the good practices in [css-style-guide](https://github.com/Wolox/tech-guides/blob/master/frontend/docs/css-style-guide.md#quotes)

File: maquetado/scss/tech.scss [hunk 3]
```diff
@@ -23,9 +23,16 @@
     margin: 15px 15px;
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]: I think a good approach is that you apply the shorthand `margin: 15px`

File: maquetado/scss/variables.scss [hunk 4]
```diff
@@ -1,12 +1,13 @@
...
-$tech-logo-size: 100px;
-$tech-height: 390px;
-$tech-width: 270px;
+$tech-back-position: -30px;
 $tech-border-radius: 5px;
+$tech-height: 390px;
 $tech-image-size: 180px;
-$tech-back-position: -30px;
+$tech-logo-size: 100px;
+$tech-transition: top 0.2s linear, left 0.2s linear;
```
Comment (henryzarza) [inline maquetado/scss/variables.scss:8]: I think that not it's necessary create a variable because you only use it once

**Review by henryzarza — CHANGES_REQUESTED** [review]

**Review by braiandickson — APPROVED** [review]

**Review by henryzarza — APPROVED** [review]

**Pull request merged**